cvat-ui/src/components/labels-editor/label-form.tsx

```diff
@@ -228,20 +264,34 @@ export default class LabelForm extends React.Component<Props> {
         const { key } = fieldInstance;
         const locked = attr.id as number >= 0;
         const existingValues = attr.values;
-
+        const { layout } = this.state;
+        const updatedLayout = layout.map((item, index) => {
+            if (index === key) {
+                return { ...item, h: 2 };
+            }
+            return item;
+        });
         const validator = (_: any, values: string[]): Promise<void> => {
             if (locked && existingValues) {
                 if (!equalArrayHead(existingValues, values)) {
+                    this.setState({ layout: updatedLayout });
                     return Promise.reject(new Error('You can only append new values'));
                 }
             }
 
             for (const value of values) {
                 if (!patterns.validateAttributeValue.pattern.test(value)) {
+                    this.setState({ layout: updatedLayout });
                     return Promise.reject(new Error(`Invalid attribute value: "${value}"`));
                 }
             }
-
+            const updatedLayout1 = layout.map((item, index) => {
```

Comment:
Lets declare a function, something like `createLayout(height)` so we dont repeat this code.
Also please, use something more meaningful than `updatedLayout` and `updatedLayout1`. defaultLayout/errorLayout